src/style.css

```diff
@@ -1,3 +1,94 @@
-body {
+html {
+  --blue-color: #00acdc;
+}
+
+/* Скидання стандартних відступів браузера */
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Встановлюємо шрифт Roboto */
+body {
+  font-family: Roboto, sans-serif;
+}
+
+/* Стилі для заголовка */
+header {
```

Comment:
better to fix it
<img width="622" alt="image" src="https://github.com/user-attachments/assets/9ff6728b-a056-462c-b8ca-a56919ec3df8">

you need to give some class to the header tag and then use that class here